frontend/controller/actions/chatroom.js

```diff
@@ -24,6 +24,9 @@ sbp('okTurtles.events/on', MESSAGE_RECEIVE_RAW, ({
   const getters = sbp('state/vuex/getters')
   const rootState = sbp('state/vuex/state')
   const targetChatroomState = rootState[contractID]
+  const currentRoute = sbp('controller/router').history.current || ''
+  const isTargetChatroomCurrentlyActive = currentRoute.path.includes('/group-chat') &&
+    getters.currentChatRoomId === contractID // when the target chatroom is currently open/active on the browser, No need to send a notification.
```

Comment:
There are two potential issues we need to consider here:

1. Should we send a notification anyway when the user is idle but on the page?
2. Are we able to, when using the service worker approach, tell what page we're on in the actions (and whether we're idle or not)? This question is for @corrideat

Reply:
I'm going to approve this after creating an issue to deal with this in a different manner later.